### PR TITLE
Create an Endpoint for migrating old ARF reports

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

File: lib/smart_proxy_openscap/openscap_import_api.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module Proxy::OpenSCAP
+  class ImportApi < ::Sinatra::Base
+    include ::Proxy::Log
+    helpers ::Proxy::Helpers
+    authorize_with_trusted_hosts
+
+    require 'smart_proxy_openscap/openscap_lib'
+
+    post "/arf/:cname/:policy_id/:date" do
+
```
Comment (ares) [inline lib/smart_proxy_openscap/openscap_import_api.rb]: nit: extra empty line

File: lib/smart_proxy_openscap/openscap_import_api.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+module Proxy::OpenSCAP
+  class ImportApi < ::Sinatra::Base
+    include ::Proxy::Log
+    helpers ::Proxy::Helpers
+    authorize_with_trusted_hosts
```
Comment (ares) [inline lib/smart_proxy_openscap/openscap_import_api.rb]: why trusted hosts instead of ssl_client like other API?

Comment (shlomizadok) [inline lib/smart_proxy_openscap/openscap_import_api.rb]: With the other API we are certain that the client is in SSL mode.
Here, Foreman can be either SSL or not SSL. (8000 / 8443)
Setting to `authorize_with_trusted_hosts` locks it only to trusted Foreman hosts

Comment (ares) [inline lib/smart_proxy_openscap/openscap_import_api.rb]: I see. If people run smart proxy on other than foreman host, they might be missing trusted host setup I think, it should be documented somewhere.

Comment (shlomizadok) [inline lib/smart_proxy_openscap/openscap_import_api.rb]: Yeah, this whole migration process needs a lot of documentation

**Comment by ares** [comment]
Merging, thanks @shlomizadok!

**Pull request merged**